webjar/src/main/webjar/pdfviewer-customizations.js

```diff
@@ -2,8 +2,29 @@
    window.addEventListener('load', function() {
       setExternalLinkTarget();
       setViewRights();
+      addCopyButton();
    }, true);
+  // #40: Add a copyable link of the PDF file location in the macro toolbar
+  function addCopyButton() {
```

Comment:
The function name is misleading. I find ``addCopyLinkButton`` or ``addPermalinkButton`` better.